apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -49,8 +77,58 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
     }
   }, [appName, dialogControl, skipUrl]);
 
+  const onClickFlash = async () => {
```

Comment:
Can we move this to a helper function? It would be good to keep the micro:bit code isolated in case we end up removing it later, and keep this component simple.

Reply:
Sure - that makes sense. Confirming that you mean a helper function from `microBit/utils.ts`?

Reply:
I don't have a strong opinion on where it should live, wherever makes sense to you!